digitalforms-api/src/main/java/ca/bc/gov/open/pssg/rsbc/digitalforms/controller/ApplicationFormController.java

```diff
@@ -0,0 +1,81 @@
+package ca.bc.gov.open.pssg.rsbc.digitalforms.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import ca.bc.gov.open.pssg.rsbc.digitalforms.model.JSONResponse;
+import ca.bc.gov.open.pssg.rsbc.digitalforms.ordsclient.DigitalFormsOrdsClientConstants;
+import ca.bc.gov.open.pssg.rsbc.digitalforms.ordsclient.application.ApplicationResponse;
+import ca.bc.gov.open.pssg.rsbc.digitalforms.model.ApplicationFormData;
+import ca.bc.gov.open.pssg.rsbc.digitalforms.service.ApplicationFormService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+
+/**
+ * 
+ * Application Form submission Controller.
+ * 
+ * @author sivakaruna
+ *
+ */
+@RestController
+@Api(value = "Application Form", tags = { "Application Form" })
+public class ApplicationFormController {
+
+	@Autowired
+	ApplicationFormService service;
+
+	@GetMapping(value = "/{formType}/application/{formGuid}", produces = "application/json")
+	@ApiOperation(value = "Get Form data", response = JSONResponse.class)
+	@ApiResponses(value = { @ApiResponse(code = 200, message = "Success", response = JSONResponse.class) })
+	public ResponseEntity<JSONResponse<ApplicationResponse>> applicationFormGet(
+			@PathVariable(value = "formType", required = true) String formType,
```

Comment:
Just a heads up that this may be changing somewhat soon. Probably won't need the form type as this will be inferred by the path use to fetch the form. No change required yet. Just wanted to tell you.